cocos/renderer/CCRenderer.cpp

```diff
@@ -321,10 +321,8 @@ void Renderer::addCommand(RenderCommand* command, int renderQueue)
     CCASSERT(!_isRendering, "Cannot add command while rendering");
     CCASSERT(renderQueue >=0, "Invalid render queue");
     CCASSERT(command->getType() != RenderCommand::Type::UNKNOWN_COMMAND, "Invalid Command Type");
-    if (command->isTransparent())
-        _transparentRenderGroups.push_back(command);
-    else
-        _renderGroups[renderQueue].push_back(command);
+    
+    _renderGroups[renderQueue].push_back(command);
```

Comment:
In real world use case 90% of the sprites are transparent, note even if there is only 1 pixel of a sprite is transparent, we have to consider it as transparent in rendering.

Also the latest graphics processor will handle opaque objects internally, so sorting on opaque objects are not necessary anymore. 
[read this](https://developer.apple.com/library/ios/documentation/3DDrawing/Conceptual/OpenGLES_ProgrammingGuide/Performance/Performance.html#//apple_ref/doc/uid/TP40008793-CH105-SW22)